.github/workflows/tests.yml

```diff
@@ -21,6 +21,9 @@ jobs:
           - macos
           - windows
         python:
+          - 'pypy-3.7'
+          - 'pypy-3.8'
+          - 'pypy-3.9'
```

Comment:
I would drop pypy-3.7 since it is not really supported and the scientific python community doesn't really support python3.7 anymore. If CI is resource constrained, it is actually probably sufficient to only run one PyPy version, perhaps 3.9.

Reply:
Ah, gotcha. I added it because the current PyPy version still supports it.